coral-spark/build.gradle

```diff
@@ -20,5 +21,5 @@ dependencies {
 }
 
 artifacts {
-  archives jar, javadocJar, sourcesJar
+  archives jar
```

Comment:
Is there any reason to remove javadocJar and sourcesJar? There are needed to publish to maven central

Reply:
Those types of jars are referenced in the main `build.gradle`. Including them here was throwing an error. I verified that they existed in the local m2 repo after the local release step.